src/dipdup/indexes/tezos_tzkt_operations/fetcher.py

```diff
@@ -123,6 +126,37 @@ async def get_origination_filters(
     return addresses, hashes
 
 
+async def get_sr_execute_filters(
+    config: TzktOperationsIndexConfig,
+) -> set[str]:
+    """Get addresses to fetch smart rollup executions from during initial synchronization"""
+    if TzktOperationType.sr_execute not in config.types:
+        return set()
+
+    addresses: set[str] = set()
+
+    if config.contracts:
+        for contract in config.contracts:
+            if contract.address:
```

Comment:
Do we need to process contracts with `code_hash` too?

Reply:
Nope, it's an unlikely situation that it would occur to anyone to deploy identical smart rollups considering how much to stake for each one.